src/Lib/Items/Model.vala

```diff
@@ -56,12 +56,7 @@ public class Akira.Lib.Items.Model : Object {
 
     private unowned ViewLayers.BaseCanvas? canvas = null;
 
-    public Model.live_model (ViewLayers.BaseCanvas? canvas) {
-        is_live = true;
-        this.canvas = canvas;
-    }
-
-    construct {
+    public Model () {
```

Comment:
I'm wondering if there's a particular reason why you converted this from a construct creation to a class creation. The `construct` always runs before any class with argument is called, so there's not need to call `this()`.
Did you stumble upon any issue with it?